docs/eks/iam.md

```
@@ -161,3 +161,116 @@ After the pods restart, describe one of the Currents pods to confirm the `AWS_WE
+**4. Confirm the Pods Picked Up the Change**
+
+Email is sent by the `server` pod (invitations) and the `writer` pod (automated
+reports). Confirm both see the SES settings and the IRSA credentials:
+
+```shell
+kubectl exec deploy/currents-writer -- env | grep -E 'EMAIL_TRANSPORTER|SES_REGION|AWS_ROLE_ARN|AWS_WEB_IDENTITY_TOKEN_FILE'
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)